src/ucp/api/ucp.h

```diff
@@ -1416,6 +1417,15 @@ typedef struct {
      * which means the memory type will be detected internally.
      */
     ucs_memory_type_t memory_type;
+
+    /**
+     * Pointer to the information where received data details are stored
+     * in case of immediate completion of receive operation.
+     */
+    union {
+        size_t              *length;
+        ucp_tag_recv_info_t *tag;
```

Comment:
variable name `tag` is very confusing. Why not `info` , seems like this is the var name used everywhere else.

Reply:
to avoid double `info`, because it is a part of `recv_info` union, so it will be `recv_info.tag_info`

Reply:
FYI, we use `info` everywhere else, so I have no issue with double naming. `tag_info` is double-double naming :) If you have strong preference towards `tag_info` I can accept it.

Reply:
fixed